README.md

```diff
@@ -1,52 +1,167 @@
 # Ajax and Callbacks
-
 ## Objectives
-+ Explain what an API is and why it's used
-+ Explain what Ajax is and why it's used
-+ Make a `get` request using Ajax to append text to a page
-+ Explain what a callback is and why Ajax runs asynchronously
+- Explain what an API is and why it's used
+- Explain what Ajax is and why it's used
+- Make a get request using Ajax to append text to a page
+- Explain what a callback is and why Ajax runs asynchronously
+
+## Intro (half done)
+
+So far, you've been using a gem to access a web API.  Gems are responsible for handling all of the nitty-gritty.  They allow us to easily access/modify the data that's important to us. But what happens if the API we want to use doesn't have an associated gem?  We will need to handle everything ourselves.  If that's the case, we better learn more about what an API really is!
```

Comment:
I'm not sure that this first sentence serves as the best introduction for the rest of the Readme. Which gem? Why have we been using a gem?

I think a simple analogy between a server-to-server call (even if we leave out that terminology) versus an Ajax call — or else something that more clearly introduces the concept of the gem's API and its analogy in the web API — would really make this introduction strong.